packages/modelviewer.dev/data/examples.json

```diff
@@ -31,19 +31,19 @@
     "examples": [
       {
         "htmlId": "demo-container-1",
-        "name": "Autoplay animation"
+        "name": "Autoplay"
```

Comment:
I think a max-width would be nice on the docs page; it's hard to read long lines. Might be nice to indent everything enough to line up with the section text instead of its box.

Reply:
done